src/app/home-page/home-logged-in/home-logged-in.component.scss

```diff
@@ -13,10 +13,17 @@ mat-sidenav-container {
   height: 100%;
 }
 
-.side-column {
+.left-side-column {
   background-color: #f6f8fa;
 }
 
+@media screen and (min-width: 992px) {
+  .side-column {
+    max-width: 350px !important;
```

Comment:
If it's max 350 and min 350, that's pretty much just width: 350px.  Additionally, I thought we banned pixels.

Reply:
What was the replacement to px?

Reply:
Used REM